src/data-service/GameData.js

```diff
@@ -0,0 +1,19 @@
+
+const GameModel = require('../model/GameDto');
+const InitialConnect = require('./SequelizeConnection');
+
+let GameDataController = {};
+
+GameDataController.createGame = async function (newGameData) {
+    return new Promise((resolve, reject) => {
+      let tempConnect = InitialConnect();      
+      let tempModel = GameModel(tempConnect);      
+      tempModel.create(newGameData).then(res => {        
+        resolve(res.Id);
+      }).catch(err => {        
+        resolve(err);
```

Comment:
we should reject something or propagate the error here, otherwise you are swallowing the exception